src/containers/MyNdla/Learningpath/components/LearningpathStepForm.tsx

```diff
@@ -55,55 +56,52 @@ interface Props {
 export const LearningpathStepForm = ({ step, stepType, onClose, onSave, onDelete }: Props) => {
   const { t } = useTranslation();
 
-  const methods = useForm<FormValues>({
-    mode: "onSubmit",
-    defaultValues: toFormValues(stepType, step),
-  });
+  const { handleSubmit, control, reset, formState } = useFormContext<FormValues>();
 
-  const { handleSubmit, control, reset, formState } = methods;
+  useEffect(() => {
+    reset(toFormValues(stepType, step));
+  }, [reset, step, stepType]);
```

Comment:
Hvorfor trengs denne?

Reply:
Formcontexten er nå global og ikke unik for hvert steg så må vi sette formValues til dataen for steget, hver gang vi åpner et steg. Dette vet vi ikke før steget er valgt.